### PR TITLE
Add login form for Crommerr signing

### PR DESCRIPTION
**Pull request by jessieay**
- Not tested at feature level because stubbing SOAP API proved difficult

screen shot 2016-01-19 at 4 35 02 pm

https://trello.com/c/HmXIrs7B

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -136,8 +136,6 @@ INSERT INTO schema_migrations (version) VALUES ('20150924181500');
 
 INSERT INTO schema_migrations (version) VALUES ('20160112190459');
 
-INSERT INTO schema_migrations (version) VALUES ('20160115181255');
```
Comment (jessieay) [inline db/structure.sql:139]: hmmm not sure why this was removed. will git checkout

File: spec/support/es_spec_helper.rb [hunk 2]
```diff
@@ -56,6 +56,8 @@ def create_es_index(klass)
     create_es_index(Manifest)
   end
   config.after :suite do
-    stop_es_server
+    if es_server_running?
```
Comment (pkarman) [inline spec/support/es_spec_helper.rb]: the check for `es_server_running?` already happens inside `stop_es_server` -- any reason you needed to check 2x?

**Comment by pkarman** [comment]
@jessieay just a minor comment. Otherwise looks great. Would be great if there was some way to reliably test/mock the SOAP stuff. Alas.

File: spec/support/es_spec_helper.rb (hunk 2 above)
Comment (jessieay) [inline spec/support/es_spec_helper.rb]: ah yes - this as an accidental re-add. removed webmock to try and test the actual CDX API responses, which was causing es problems. removed at c2a6d24

**Comment by jessieay** [comment]
@pkarman this is ready for re-review

**Pull request merged**